### ISSUE TITLE
The wallpaper stops after some time, when using programs in maximized mode, but not un-stop's when you look at them

### ISSUE THREAD

**Issue by Glesas** [jeffshee/gnome-ext-hanabi#21]
For now, when i using program in maximized mode (not a fullscreen) the wallpaper seem to pausing itself, after some time. But when i going on desktop itself, the wallpapers is not playing again, and i have to re-enable extension again, just to see wallpapers work again

My suggestion is: maybe make the switch / command attribute, that will let wallpapers run always, no matter what? (Well yes, higher resource usage, but i think this is easiest thing to create).  Or idk, make the hotkey that will let re-enable extension if that happens

**Comment by jeffshee** [comment]
Hi @Glesas 

> For now, when i using program in maximized mode (not a fullscreen) the wallpaper seem to pausing itself, after some time.

The auto pause feature you described does not exist in this extension, yet. So what you observed may be the wallpaper itself froze for some reason, and I can't seem to reproduce the issue on my system (I tried with maximized Firefox, and no issue so far).

Please tell me more about your system and also the steps to reproduce the issue.

- OS Distribution:
- GNOME version:
- X11/Wayland:
- Steps to reproduce the issue:
  1.
  2.
  
Also, please try with another video and see if the issue remains.
Thanks!

**Comment by daxime** [comment]
Hi,
i have the same issue. It happen after few hours, but not specifically when a window is maximized.

I installed the extension yesterday. I usually leave a video running in full screen when I go to bed. When I woke up, the wallpaper was frozen.

A few minutes ago, the background froze again for the 2nd time today, I had several windows open, but not full screen

I use Manjaro (up-to-date), Gnome 43.3, Wayland

I also noticed that the video is slightly blurry, when I watch the video in my video player in full screen, the image is sharp. The video is 1080p and my screen resolution is also 1080p.

PS: Thank you for your extension, I've been looking for an animated wallpaper for 2 years and your extension is the only solution that worked for me.
